source/javascripts/index.js

```diff
@@ -1,34 +1,38 @@
-import React from 'react';
-import { createRoot } from 'react-dom/client';
-import 'bootstrap/dist/js/bootstrap';
+import React from "react";
```

Comment:
The quote changes are distracting, can we leave them out?

Thinking about it, this is the `develop` branch and as you're the only one currently working on it we can rewrite the history and cleanup the un-needed/temporary commits.

I don't remember what convention this project has for quotes but if it's to use double quotes let's make a different PR for it.

Reply:
this is my prettier settings, i can switch back to single quote, but for vanilla javascript is more common double quotes.

Reply:
Let's at least put those changes in a different commit.

Reply:
ok. i'll try